src/main/java/com/starkinfra/DynamicBrcode.java

```diff
@@ -0,0 +1,587 @@
+     * Send a list of DynamicBrcode objects for creation in the Stark Infra API
+     * <p>
+     * Parameters:
+     * @param brcodes [list of DynamicBrcode objects or HashMaps]: list of DynamicBrcode objects to be created in the API.
+     * <p>
+     * Return:
+     * @return list of DynamicBrcode objects with updated attributes
+     * @throws Exception error in the request
+     */
+    public static List<DynamicBrcode> create(List<?> brcodes) throws Exception {
+        return DynamicBrcode.create(brcodes, null);
+    }
+
+    /**
+     * Create DynamicBrcodes
+     * <p>
+     * Send a list of DynamicBrcode objects for creation in the Stark Infra API
+     * <p>
+     * Parameters:
+     * @param brcodes [list of DynamicBrcode objects or HashMaps]: list of DynamicBrcode objects to be created in the API.
+     * @param user [Organization/Project object, default null]: Organization or Project object. Not necessary if starkinfra.Settings.user was set before function call
+     * <p>
+     * Return:
+     * @return list of DynamicBrcode objects with updated attributes
+     * @throws Exception error in the request
+     */
+    @SuppressWarnings("unchecked")
+    public static List<DynamicBrcode> create(List<?> brcodes, User user) throws Exception {
+        List<DynamicBrcode> brcodeList = new ArrayList<>();
+        for (Object brcode : brcodes){
+            if (brcode instanceof Map){
+                brcodeList.add(new DynamicBrcode((Map<String, Object>) brcode));
+                continue;
+            }
+            if (brcode instanceof DynamicBrcode){
+                brcodeList.add((DynamicBrcode) brcode);
+                continue;
+            }
+            throw new Exception("Unknown type \"" + brcode.getClass() + "\", use DynamicBrcode or HashMap");
+        }
+        return Rest.post(data, brcodeList, user);
+    }
+
+    /**
+     * Helps you respond to a due DynamicBrcode Read
+     * <p>
+     * When a Due DynamicBrcode is read by your user, a GET request containing the BR Code's
+     * UUID will be made to your registered URL to retrieve additional information needed
+     * to complete the transaction.
+     * The get request must be answered in the following format, within 5 seconds, and with
+     * an HTTP status code 200.
+     * <p>
+     * Parameters:
+     * version [integer]: integer that represents how many times the BR code was updated.
+     * created [string]: creation datetime in ISO format of the DynamicBrcode. ex: "2022-05-17"
+     * due [string]: requested payment due datetime in ISO format. ex: "2022-05-17"
+     * keyId [string]: receiver's PixKey id. Can be a taxId (CPF/CNPJ), a phone number, an email or an alphanumeric sequence (EVP). ex: "+5511989898989"
+     * status [string]: BR code status. Options: "created", "overdue", "paid", "canceled" or "expired"
+     * reconciliationId [string]: id to be used for conciliation of the resulting Pix transaction. ex: "cd65c78aeb6543eaaa0170f68bd741ee"
```

Comment:
Especifica aqui:
`This id must have from to 26 to 35 alphanumeric characters ex: "cd65c78aeb6543eaaa0170f68bd741ee"`